test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -353,7 +353,7 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		const selectorAriaLabel = ariaLabel || `Block: ${ title }`;
 
 		const inserterBlockItemSelector = By.css(
-			`.edit-post-layout__inserter-panel .block-editor-inserter__block-list button.editor-block-list-item-${ prefix }${ blockClass }`
+			`.edit-post-layout__inserter-panel .block-editor-block-types-list button.editor-block-list-item-${ prefix }${ blockClass }`
```

Comment:
Probably related to the change that led to this failure, the search results no longer seem to show the blog category. See the example of the "Dynamic HR" block:

| GB 9.3 | GB 9.4 RC1 |
| ------------- | ------------- |
| ![gb93](https://user-images.githubusercontent.com/81248/99472325-fd8b2380-290d-11eb-9e71-7090c5cd1ff2.png) | Content Cell
![gb94](https://user-images.githubusercontent.com/81248/99472331-00861400-290e-11eb-99b0-3b93d442c039.png) |

I'm pretty sure this is just an UX change and not a bug, but taking a note here in case it causes confusion down the road.